src/app/core/parsers/zod-entity-parser.service.ts

```diff
@@ -6,8 +6,38 @@ import { Rundown } from '../models/rundown'
 import { Segment } from '../models/segment'
 import * as zod from 'zod'
 import { PieceType } from '../enums/piece-type'
+import { ShowStyleVariant } from "../models/show-style-variant";
 
 export class ZodEntityParser implements EntityParser {
+    private readonly blueprintConfigurationParser = zod.object( {
+        SelectedGfxSetupName: zod.object( {
+            value: zod.string(),
+            label: zod.string(),
+        }),
+        GfxDefaults: zod.object({
+            id: zod.optional(zod.string()),
+            DefaultSetupName: zod.object({
+                value: zod.string(),
+                label: zod.string(),
+            }),
+            DefaultSchema: zod.object({
+                value: zod.string(),
+                label: zod.string(),
+            }),
+            DefaultDesign: zod.object({
+                value: zod.string(),
+                label: zod.string(),
+            }),
+        }).array()
+    })
```

Comment:
Perhaps it should just be explained in the README.md, such than any other company that wishes to fork it, knows that they will need to adjust certain parts to their Blueprints?

Reply:
It's a good point that we should discuss how we handle this, whether it be a more specific layer put on top so it can easily be recognized as specific to our own implementation or at least described somewhere.